.github/workflows/template.flink-ci.yml

```diff
@@ -316,7 +316,7 @@ jobs:
           maven_repo_folder: ${{ env.MAVEN_REPO_FOLDER }}
 
       - name: "Install missing packages"
-        run: sudo apt-get install -y net-tools docker-compose zip
+        run: sudo apt-get install -y net-tools docker compose zip
```

Comment:
is this right. We are installing docker-compose not using the command line docker-compose. l suspect that we want to remove the word `compose`; as we are not installing something called compose. I suggest
`run: sudo apt-get install -y net-tools docker zip`

Reply:
how can I test it?

Reply:
fixed